lib/cli/package.json

```diff
@@ -52,10 +52,10 @@
     "fs-extra": "^9.0.0",
     "get-port": "^5.1.1",
     "globby": "^11.0.0",
-    "inquirer": "^7.0.0",
     "jscodeshift": "^0.6.3",
     "json5": "^2.1.1",
     "leven": "^3.1.0",
+    "prompts": "^2.0.0",
```

Comment:
You need to add `@types/prompts` to devDep as you're using `prompts` in a TS file in `lib/cli/src/initiate.ts`